src/VisualStudio/SetupInteractive/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="A5E5EE8D-39AE-42FF-8BBF-53C5B09C07D7" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Interactive Language Services</DisplayName>
     <Description>Roslyn interactive language services.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\InteractiveLanguageServices</ShortcutPath>
```

Comment:
Just curious - Why aren't we ngen-ing the Repl binaries in this VSIX? (Were they being ngen-ed in VS 2015?)